templates/projects/snippets/relevancy.html

```diff
@@ -0,0 +1,79 @@
+<div id="task-app">
+  <task-presenter>
+  <div class="container-fluid">
+      <div class="row">
+        <blockquote>
+
+        <h4>Rating Topic Relevancy</h4>
+        <p>This template may be used to rate the relevancy of queries against a topic. The user is presented with a list of queries and can choose "relevant" or "irrelevant" with regard to the topic.</p>
+
+        <h5>Input</h5>
+        <p>Tasks can be imported by creating a CSV file containing 2 columns: topic, queries.</p>
+        <p>The field "topic" represents the subject for the particular task to rate the relevancy upon. The field "queries" is a comma-separated string of queries, from which the user may rate the relevancy against each topic.</p>
+
+        <h5>Output</h5>
+        <p>The output from the task will contain a response of "relevant" or "irrelevant" for each query, with regard to the topic.</p>
+</blockquote>
+
+        <p>Below you will find search results from a query. Please rate each search result as either relevant or irrelevant, according to the topic displayed.</p>
+      </div>
+      <div class="row">
+          <h2>Topic: {% raw %}{{task.info.topic}}{% endraw %}</h2>
```

Comment:
I think this is not the right way to handle the issue we noticed yesterday, since when this template is shown on the UI it shows {% raw %} 
![Screen Shot 2019-07-12 at 10 58 50 AM](https://user-images.githubusercontent.com/7351421/61137770-225ac580-a494-11e9-919f-470bee90a9ae.png)

Reply:
Unless Im missing something

Reply:
I'm not seeing that on my end. Could you confirm the template?

Reply:
Never mind It was my mistake